### PR TITLE
Circleci integration

### PR DESCRIPTION
**Pull request by wilzbach**
CircleCI integration (#101).
It seems to be pretty fast (<30s)!

### CODE REVIEW

**Comment by wilzbach** [comment]
See https://circleci.com/gh/wilzbach/mir

When you enable it via the web interface - there is also a setting to allow up to 4 parallel builds :)

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+machine:
+  environment:
+    DMD: 2.071.0
+    DUB: 0.9.24
+    PATH: "${HOME}/dmd2/linux/bin64:${PATH}"
+    LD_LIBRARY_PATH: "${HOME}/dmd2/linux/lib64:${LD_LIBRARY_PATH}"
```
Comment (9il) [inline circle.yml:6]: OS X support?

Comment (wilzbach) [inline circle.yml:6]: Not possible in the free version - https://circleci.com/pricing/#build-linux

Comment (9il) [inline circle.yml:6]: oh, ok

**Comment by wilzbach** [comment]
Btw at some point we can also setup a [build matrix](https://github.com/michaelcontento/circleci-matrix) there, but it seems to be more complicated and for now we can enjoy the fast success / error from within one minute :)

**Pull request merged**